src/components/TaskHeader.js

```diff
@@ -60,7 +59,7 @@ function TaskHeader(props) {
                     >
                         <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween, styles.pv3]}>
                             <View style={[styles.flexRow, styles.alignItemsCenter, styles.justifyContentBetween]}>
-                                {!_.isEmpty(props.report.managerID) && (
+                                {props.report.managerID > 0 && (
```

Comment:
I've updated this to add `>0` as it was causing this weird bug

<img width="438" alt="Screenshot 2023-06-15 at 17 13 38" src="https://github.com/Expensify/App/assets/9680864/565bb971-2bed-497b-8e5e-6d83998715fb">

Reply:
Let's also check if this exists first props.report.managerID &&